packages/forma-36-react-components/src/components/Card/AssetCard/AssetCard.tsx

```diff
@@ -47,17 +44,13 @@ export type AssetCardProps = {
    */
   type?: AssetType;
   /**
-   * Renders a drag handle for the component for use in drag and drop contexts
+   * Prop to pass a CardDragHandle component to for use in drag and drop contexts
    */
-  withDragHandle?: boolean;
+  cardDragHandleComponent?: React.ReactNode;
```

Comment:
Isn't it a breaking change?

Reply:
Yeah sadly, it's flagged in the commit message so should be picked up in release notes / version bump.

Reply:
I'd rather leave `withDragHandle` method and add ability to use custom component when needed. By default, `CardDragHandle` should be used.

Reply:
@suevalov I like this approach as it'll save us from making a breaking change, however, the drag handles are currently pretty useless without this change and the `cardDragHandleComponent` would be the prop used in pretty much all cases.

Reply:
Just to clarify to anyone reading - went with Alex's solution, we use both props, so no breaking changes